src/components/bulk-email-tool/bulk-email-form/data/api.js

```diff
@@ -1,8 +1,20 @@
 /* eslint-disable import/prefer-default-export */
 import { getConfig } from '@edx/frontend-platform';
 import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
+import { logError } from '@edx/frontend-platform/logging';
 
-export async function postBulkEmail(email, courseId) {
+export async function postBulkEmailInstructorTask(email, courseId) {
   const url = `${getConfig().LMS_BASE_URL}/courses/${courseId}/instructor/api/send_email`;
   return getAuthenticatedHttpClient().post(url, email);
 }
+
+export async function patchScheduledBulkEmailInstructorTask(emailData, courseId, taskId) {
+  const endpointUrl = `${getConfig().LMS_BASE_URL}/api/instructor_task/v1/schedules/${courseId}/bulk_email/${taskId}`;
```

Comment:
Nit: Do you think it would be better to rename `taskId` to `scheduleId`? Since the item we are updating is the actual instructor task schedule, and not the instructor task. Or, maybe we could write a comment about what the expected value of `taskId` represents?

That might reduce potential confusion down the road.

Reply:
changed